content/webapp/pages/search/index.tsx

```diff
@@ -339,12 +455,14 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
                           `images_all_${pathname}`
                         )}
                         passHref
+                        legacyBehavior
                       >
-                        All images {catalogueResults.images?.totalResults}
+                        <AllLink type="images" results={catalogueResults} />
                       </NextLink>
```

Comment:
Both links are displaying the same numbers 
<img width="449" alt="Screenshot 2025-02-03 at 12 06 07" src="https://github.com/user-attachments/assets/a5f7e09e-9e4f-45b7-8c44-5665ce021c9d" />

Reply:
nice catch, thanks